src/WinGetMCPServer/WingetPackageTools.cs

```diff
@@ -0,0 +1,266 @@
+// -----------------------------------------------------------------------------
+// <copyright file="WingetPackageTools.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace WinGetMCPServer
+{
+    using Microsoft.Management.Deployment;
+    using ModelContextProtocol.Protocol;
+    using ModelContextProtocol.Server;
+    using System.ComponentModel;

```

Comment:
nit: I guess Stylecop is not turned on for this project? It should complain that System. should be on the top.